tests/unit/test_synapse_workload.py

```diff
@@ -129,11 +129,12 @@ def test_enable_ip_range_whitelist_success(ip_range_whitelist: str, harness: Har
 
     with open(config_path, encoding="utf-8") as config_file:
         content = yaml.safe_load(config_file)
+        expected_ip_range_whitelist = [item.strip() for item in ip_range_whitelist.split(",")]
```

Comment:
I would suggest making this a hardcoded value in the param since this is essentially doing the same thing as the code. For example:

```python
@pytest.mark.parametrize(
    "ip_range_whitelist, expected_ip_range_whitelist",
    [
...
        pytest.param(
            "10.10.10.10 ",
            ["10.10.10.10"],
            id="single_item_trailing_whitespace",
            marks=pytest.mark.xfail(strict=True),
        ),
...
    ],
)
def test_enable_ip_range_whitelist_success(ip_range_whitelist: str, expected_ip_range_whitelist: list[str], harness: Harness):
```

That way we clearly state the input and expected output for each test and avoid accidental bugs in the test code leading to tests unexpectedly passing

Reply:
Thanks,I'll modify that and for following tests in this PR that is related to this configuration
https://github.com/canonical/synapse-operator/pull/119